tests/pairlist/test_pairlist.py

```diff
@@ -302,6 +302,10 @@ def test_VolumePairList_refresh_empty(mocker, markets_empty, whitelist_conf):
     ([{"method": "VolumePairList", "number_assets": 5, "sort_key": "quoteVolume"},
       {"method": "ShuffleFilter"}],
      "USDT", 3),  # whitelist_result is integer -- check only length of randomized pairlist
+    # PerformanceFilter
+    ([{"method": "VolumePairList", "number_assets": 5, "sort_key": "quoteVolume"},
+      {"method": "PerformanceFilter"}],
+     "USDT", ['ETH/USDT', 'NANO/USDT', 'ADADOUBLE/USDT', 'ADAHALF/USDT']),
```

Comment:
@xmatthias ... continued from [this thread](https://github.com/freqtrade/freqtrade/pull/4009#discussion_r532053809):

This doesn't solve the issue unfortunately - at least when there aren't yet trades.

I committed and pushed the test case with incoming list of `['ETH/USDT', 'NANO/USDT', 'ADADOUBLE/USDT', 'ADAHALF/USDT']` still getting rearranged to `['ETH/USDT', 'NANO/USDT', 'ADAHALF/USDT', 'ADADOUBLE/USDT']` - presumably due sorting happening within to `VolumePairList` or elsewhere.

Reply:
I restored the order to enable tests to pass. I am convinced that any issue here lies outside the scope of this PR.

Reply:
well ideally, the result will be identical with / without Performancefilter (only with the filters before it) - as in case of "No trades yet" - we'll simply skip this filter as it will not work anyway.

Reply:
Test case now removed.